style.css

```diff
@@ -1,4 +1,53 @@
 * {
     margin: 0;
```

Comment:
seria bom resetar o padding tb.
```suggestion
    margin: 0;
    padding: 0;
```